README.md

```diff
@@ -1,39 +1,37 @@
-<!--
-This README describes the package. If you publish this package to pub.dev,
-this README's contents appear on the landing page for your package.
+# Flutter Introduction
 
```

Comment:
add the gif in the markdown to so people can instantly see how the component looks when opening the page